frontend/public/components/_project-overview.scss

```diff
@@ -114,4 +135,48 @@
       margin-bottom: 20px;
     }
   }
+
+  @media (min-width: $screen-md-min) {
+    // Override Patternfly list view styles so that the additional info has a
+    // little more space for alerts, metrics, and pod status.
+    .list-view-pf-additional-info {
```

Comment:
should this be nested in `.project-overview` so that we aren't overriding all `.list-view-pf-additional-info` styles?

Reply:
It is if you look above. (Not obvious looking at the diff...)

Reply:
I'm the one that did that too... 😆